src/less/button/ds6/button-base.less

```diff
@@ -4,6 +4,11 @@
 // so all font-sizes are defined with px
 
 // base properties for all buttons
+button:not([class*="btn--"]) {
```

Comment:
@seangates Why are we adding a global rule? Which issue does this address? If there is a legitimate need for this it should go in global module.

Reply:
Let me adjust this. You're right that it shouldn't be global